src/main/java/uk/gov/hmcts/divorce/divorcecase/model/access/AosAccess.java

```diff
@@ -21,6 +22,7 @@
         grants.putAll(LEGAL_ADVISOR, Permissions.READ);
         grants.putAll(APPLICANT_2_SOLICITOR, Permissions.CREATE_READ_UPDATE);
         grants.putAll(APPLICANT_2, Permissions.CREATE_READ_UPDATE);
```

Comment:
Maybe applicant2 role can be removed?

Reply:
Yeah I am not sure why Somesh added it originally